File: JavaScript/010 - Summation of primes/solution.js
var prime = require('../shared/prime');

function execute() {
    var sum = 2;
    for (var i = 3; i < 2e+6; i += 2) {
        if (prime.isPrime(i)) {
            sum += i;
        };
    }
    return sum;
}
module.exports.execute = execute;

File: GoLang/010 - Summation of primes/solution.go
package project_euler

import (
	"../shared"
)

func execute010() int {
	sum := 2
	for i := 3; i < 2e+6; i += 2 {
		if shared.IsPrime(i) {
			sum += i
		}
	}
	return sum
}
